src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const GROUP_STYLE = sourceString.trim().split(';');
+  const GET_CLEAN_STYLE = GROUP_STYLE.map((pair) => pair.trim()).filter(
+    (style) => style !== '',
+  );
```

Comment:
The variable 'GET_CLEAN_STYLE' should be named to reflect its purpose more clearly, like 'cleanedStylePairs'. Additionally, the chaining of map and filter can be indented to improve readability.